src/media/AvatarMember.jsx

```diff
@@ -39,20 +34,24 @@ class AvatarMember extends React.PureComponent {
 		return (
 			<Avatar
 				alt={member.name}
-				src={showNoPhoto ? noPhotoImage : member.photo.photo_link}
+				src={showNoPhoto ? noPhotoImage : member.photo[photoLink]}
 				className={classNames}
+				big={big}
```

Comment:
we shouldn't need to specify these as they _should_ come through with `...other` did you observe something different?

Reply:
Yeah I saw something different. They don't come through because of the destructuring statement `const { ... , big, large, xxlarge, ...other } = this.props;`. There's a more clear example here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

Reply:
But I could do something like:

```
const { member, org, fbFriend, className, ...other } = this.props;
const { big, large, xxlarge } = other;
```
How does that feel to you?

Reply:
yes yes your right! leave as is